examples/pytorch/eeg-gcnn/README.md

```diff
@@ -0,0 +1,52 @@
+# DGL Implementation of EEG-GCNN Paper
+This DGL example implements the EEG-GCNN model proposed in the paper [EEG-GCNN](https://arxiv.org/abs/2011.12107). The original code is in [here](https://github.com/neerajwagh/eeg-gcnn).
+
+## All References
+- ArXiv Pre-print: <https://arxiv.org/abs/2011.12107>
+- PMLR Paper: <http://proceedings.mlr.press/v136/wagh20a.html>
+- ML4H Poster: <https://drive.google.com/file/d/14nuAQKiIud3p6-c8r9WLV2tAvCyRwRev/view?usp=sharing>
+- ML4H 10-minute Video: <https://slideslive.com/38941020/eeggcnn-augmenting-electroencephalogrambased-neurological-disease-diagnosis-using-a-domainguided-graph-convolutional-neural-network?ref=account-folder-62123-folders>
+- ML4H Slides: <https://drive.google.com/file/d/1dXT4QAUXKauf7CAkhrVyhR2PFUsNh4b8/view?usp=sharing>
+- Code: [GitHub Repo](https://github.com/neerajwagh/eeg-gcnn)
+- Final Models, Pre-computed Features, Training Metadata: [FigShare .zip](https://figshare.com/articles/software/EEG-GCNN_Supporting_Resources_for_Reproducibility/13251452)
+- Raw Data: [MPI LEMON](http://fcon_1000.projects.nitrc.org/indi/retro/MPI_LEMON.html) (no registration needed), [TUH EEG Abnormal Corpus](https://www.isip.piconepress.com/projects/tuh_eeg/downloads/tuh_eeg_abnormal/) ([needs registration](https://www.isip.piconepress.com/projects/tuh_eeg/html/request_access.php))
```

Comment:
- I think we can omit the item for "ArXiv Pre-print" and "Code" as they were covered in the section above.
- Can you add an explanation for the rest items so people can understand what they are and in what cases they can be helpful?